weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage.tsx

```diff
@@ -722,3 +696,137 @@ const useCallFlattenedTraceTree = (call: WFCall) => {
     return {rows, expandKeys};
   }, [call]);
 };
+
+const CallDetails: FC<{
```

Comment:
I'd like to recommend we start putting components with more than a few lines into their own files, I think it helps with readability, searching within files, etc.

Reply:
Done